saleor/payment/gateways/adyen/tests/test_plugin.py

```diff
@@ -719,5 +721,8 @@ def test_adyen_check_payment_timeout(adyen_plugin, adyen_check_balance_response)
         },
     }
 
-    with pytest.raises(requests.exceptions.ConnectTimeout):
-        plugin.check_payment_balance(data)
+    # with pytest.raises(requests.exceptions.ConnectTimeout):
```

Comment:
You could leave `with pytest.raises(requests.exceptions.ConnectTimeout):` but in `#given`, you could set `request_post_mock.side_effect = ConnectTimeout`.
Also, just thought about this: I think that we should capture this error on plugin side and convert it to `PaymentError`, as we did for the rest errors 👇 

```
def api_call(
    request_data: Optional[Dict[str, Any]], method: Callable, **kwargs
) -> Adyen.Adyen:
    try:
        return method(request_data, **kwargs)
    except (Adyen.AdyenError, ValueError, TypeError) as e:
        logger.warning(f"Unable to process the payment: {e}")
        raise PaymentError(f"Unable to process the payment request: {e}.")
```